articleEditor/editor.js

```diff
@@ -13,10 +13,11 @@
         }
     });
 
-    ShowArticles(window.articlesFromLS);
-    CarouselMaket(window.articlesFromLS);
+    window.card.Show(window.articlesFromLS);
```

Comment:
This method should receive as parameter only `ids` of articles to show them.
Inside, you can pass through articles array from localstorage.